lib/src/actions/package/providers/paru.rs

```diff
@@ -92,7 +92,7 @@ impl PackageProvider for Paru {
 
     fn query(&self, package: &PackageVariant) -> anyhow::Result<Vec<String>> {
         let requested_already_installed: HashSet<String> = String::from_utf8(
-            Command::new("paru")
+            Command::new("pacman")
```

Comment:
Could we use match to check first if paru is installed, if not, then fall back to pacman, otherwise use paru. We could use lib::utilities::get_binary_path, on error fall back to pacman.

Reply:
Since we're just checking that a package installed, does it really matter which binary we use?

Reply:
Yea, that makes sense. This is also a blind spot for me since I've not used paru or yay. So if a package is installed with yay or paru, pacman can still pick it up with a query then?

Reply:
Yeah. Tested it and I am using a build from this PR right now :)